openfermioncirq/primitives/optimal_givens_decomposition.py

```diff
@@ -0,0 +1,132 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A routine for constructing a circuit to exactly implement a unitary generated by
```

Comment:
I feel that's also fair to mention our PRL here. Its not totally trivial that this idea carries over from the optics context into the fermion context (otherwise the optics paper would mention that!).

Reply:
💯